TeamCode/src/main/java/org/firstinspires/ftc/teamcode/stateslist.java

```diff
@@ -134,7 +198,7 @@ public void onCompletion() {
     /***
      * state makes robot use color sensor and servo to try to press the button on the beacon
      */
-    state pushBeaconButton = new state("use servo to select color") {
+    state pushBeaconButton = new state("driveTowardsBeacon") {
```

Comment:
The name of this state is incorrect.
`state pushBeaconButton = new state("driveTowardsBeacon") { `